test/verify/check-system-services

```diff
@@ -34,6 +34,7 @@ class TestServices(MachineCase):
         # when we add additional services to this target below
         # we don't race with the boot process
         self.machine.execute("systemctl start default.target")
+        self.machine.execute("systemctl --user start default.target")
```

Comment:
I'm quite surprised that this is necessary. Do we have some evidence that it is not already running after logging into cockpit? Unlike the system instance, this should do pretty much nothing. I'm a bit afraid of hiding errors here if it's not at least in "activating". And I suppose we should change the other command from an explicit "start" (which hides bugs) to a waiting loop on `is-active`.